### DISCUSSION TITLE
steep paths like rct3 and parkitect

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17612] [Feature requests]
If there were a way to build paths that allowed guests to go up twice as fast that would be great. I assume it is possible to program into the game a guest traversing 2 path heights high, and path could just be a grey or black square. I don't think it would be difficult to implement but I may be wrong

**Comment by duncanspumpkin** [comment]
This would be very very hard to implement and would require all path makers to add the new path sprites.

**Reply by 733737** [reply]
what do you mean by path makers?

**Reply by Broxzier** [reply]
People who created the path objects.

**Reply by ocalhoun6** [reply]
Maybe there could be a 'stairs' object, kind of like a scenery object, but able to connect to paths and with a steeper slope. This object would be its own separate thing and not be designed to match any of the existing path types. Though it might still come in a few different styles like 'wooden stairs', 'concrete stairs', and 'metal stairs'.

Since the vanilla path types don't include matching stairs, it wouldn't be expected for new path types to include matching stairs either. Though the stairs objects should still support new designs made by users.

Something would have to be added to the path building UI to let people select the steep stairs, though, and to let them select which style to use.

------

That said, I don't know if it's _that_ much of a burden on path object designers to also include steep path sprites. That's what ... 4 sprites? Doesn't seem like _that_ big of a deal when you're already making a whole bunch of sprites for all the other types of path tiles.

**Reply by X123M3-256** [reply]
> Maybe there could be a 'stairs' object, kind of like a scenery object, but able to connect to paths and with a steeper slope. 

This sounds a lot harder to implement than just adding steep slopes to the paths. You'd need to add like 4 sprites per path object and they could be optional (not all wall objects have the sprites for slopes, for example). I'm sure we could find some people to create a few new sprites for the vanilla paths and custom paths not necessarily having them shouldn't be a barrier to adding new features IMO.

Personally, though, I'd much rather see a half slope than a steeper one.

**Reply by ocalhoun6** [reply]
If you provide half slopes, then people will build accessible rides at half heights ... sounds fine at first ... but then maybe you want your ride to have two stations and connect the other station to a normal height path.

So now, all tracked rides could really use a half slope track piece to accomplish this, especially transport rides.

Could be opening a can of worms.

**Reply by 733737** [reply]
I think starting with 1 or 2 path types getting steep path (IE dirt and tarmac) would be good, and could be used as placeholder until there's more. After all, it's just a flat surface in isometric from SW, NW, SE, NE, in fact I could make the objects myself, I have made objects before...
